flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/plan/rules/dataSet/DataSetUnionRule.scala

```diff
@@ -32,6 +32,13 @@ class DataSetUnionRule
       "FlinkUnionRule")
   {
 
+  /**
+   * Only translate UNION ALL
+   */
+  override def matches(call: RelOptRuleCall): Boolean = {
+    val union: LogicalUnion = call.rel(0).asInstanceOf[LogicalUnion]
+    union.all
+  }
     def convert(rel: RelNode): RelNode = {
```

Comment:
Add a newline between the methods. I also looks like the indention of {{convert}} is a bit messed up. Can you correct that as well?